tools/disasm/functions.txt

```diff
@@ -8570,7 +8570,7 @@
     0x8099A920:("ObjSound_Init",),
     0x8099A96C:("ObjSound_Destroy",),
     0x8099A9A4:("ObjSound_Update",),
-    0x8099AA84:("func_8099AA84",),
+    0x8099AA84:("ObjSound_Draw",),
```

Comment:
Huh? this actor c code wasn't touched by this PR I don't think?

I guess it's possible that the function is a draw func... although I thought obj_sound was supposed to be a source of sound without anything visible, like fountain sfx and stuff.

but we can't see it in this PR, can I assume the actor is already decompiled and this was just from trying to restrict the PR size/scope?

Reply:
`ObjSound_Draw` was decompiled in the very last audio PR I did. Since everything is on a main audio branch, I realized I just missed this rename here. It's already named and in the repo, this is just catching up this miss to my main audio branch